CONTRIBUTING.rst

```diff
@@ -121,16 +136,52 @@ Ready to contribute? Here's how to set up `renku` for local development.
 
 7. Submit a pull request through the GitHub website.
 
+
+Commit message guidelines
+-------------------------
+
+This project is using conventional_ commits style for generation of changelog upon
+each release. Therefore, it's important that our commit messages convey what they
+do correctly. Commit message should always follow this pattern:
+
+.. _conventional: https://www.conventionalcommits.org/en/v1.0.0/
+
+   $ %{type}(%{scope}): %{description}
+
+**Type's used for describing commit's which will end up in changelog are** :code:`fix:` & :code:`feat:`.
+
+Please note that the :code:`fix` type here is only for user-facing bug fixes and not fixes on tests or CI.
+For those, please use: :code:`ci:` or :code:`test:`
+
+Full list of types which are in use:
+  * :code:`feat:` - Used for new user-facing features. This should be related to one of the predefined scopes. If a scope does not exist, a new scope may be proposed.
+  * :code:`fix:` - Used for fixing user-facing bugs. This should be related to one of the predefined scopes.
+  * :code:`chore:` - Used for changes which are not user-facing. The scope should be a module name in which chore occurred.
+  * :code:`tests:` - Used for fixing existing or adding new tests. The scope should relate to a module name which is being tested.
+  * :code:`docs:` - Used for adding more documentation. If documentation is not related to predefined user scopes, it can be omitted.
+  * :code:`refactor` - Used for changing the code structure. Scope being used here should be module name. If refactoring is across multiple modules, scope could be omitted or PR broken down into smaller chunks.
+
+Full list of user-facing scopes which are in use:
+  * :code:`graph` - Scope for describing knowledge graph which is being build with users usage of the system.
+  * :code:`workflow` - Scope for describing reproducibility flow.
+  * :code:`dataset` - Scope for describing datasets.
+  * :code:`core` - General scope for describing all core elements of Renku project.
+  * :code:`svc` - General scope for describing interaction or operation of Renku as a service.
```

Comment:
I prefer `service` instead of `svc`.

Reply:
At the risk of things being more verbose, I second that